### DISCUSSION TITLE
Read aloud is "skipping" through pages without media overlay implementation

### DISCUSSION THREAD

**Discussion by brianagina** [edrlab/thorium-reader#1754] [Ideas]
When the read aloud is activated, pages without media overlay implementation in the reading order are "skipped". This presents an issue in EPUB publications with pages that have and do not have media overlay implementation in the reading order. Each page without media overlay implementation is "skipped" through, barely few seconds after being opened. It is challenging to stop the read aloud when the pages are being "skipped". 
- Can we have the thorium-reader stop read aloud when on a page without media overlay implementation so that the page is not "skipped" through?
- Is there a work around in the EPUB code?

**Comment by danielweck** [comment]
Hello, this is actually a feature, not a bug. The previous behaviour was to quickly skip non-narrated pages, but this was problematic in certain cases such as illustrated children's books with visual spreads adjacent to textual / spoken content.
Do you know how other reading systems present EPUB3 Media Overlays that have narration "holes" in them? Do they ignore the documents by skipping them in the reading order? I remember Apple iBooks pausing for 2s on non-narrated pages, but this was FXL and their implementation may have changed since...

**Comment by brianagina** [comment]
_Do you know how other reading systems present EPUB3 Media Overlays that have narration "holes" in them?_ 
- Quite similar behavior  .The discontinued Adobe Digital Editions and iBooks also skip through non-narrated pages. I had a similar experience on iBooks having tested recently. 
When can this feature be part of the roadmap?

**Comment by danielweck** [comment]
Moving this feature discussion to a ... discussion :)
